React/Views/ScrollView/RCTScrollView.m

```diff
@@ -941,8 +947,6 @@ - (void)scrollViewDidScroll:(RCTCustomScrollView *)scrollView // [macOS]
   }
 #if !TARGET_OS_OSX // [macOS]
   RCT_FORWARD_SCROLL_EVENT(scrollViewDidScroll : scrollView);
-#else // [macOS
-  (void) scrollView;
```

Comment:
This condition was here since the beginning of RN Desktop. Removing since it doesn't seem to do anything.

Reply:
Heh... This feels like a weird incomplete patch of code... Thanks for the catch.

Reply:
Do we still need the macOS ifdefs? Also can we update the closing tag to have both brackets?